stock_available/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2014 Numérigraphe
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock available to promise',
+    'version': '12.0.1.0.0',
```

Comment:
@devang-dreambits Add website key and URL like `https://github.com/OCA/stock-logistics-warehouse`